plugins/datadog/app/models/datadog_monitor.rb

```diff
@@ -73,19 +73,10 @@ def reload
 
   private
 
-  # alerting scopes, but ignore when it completely matches a silenced scope
   # @return [Array<String>]
   def alerting_tags
-    alerting = tags_from_response(:state, :groups)
-    silenced = tags_from_response(:options, :silenced)
-    alerting.reject! { |scope| silenced.any? { |s| (s & scope) == s } }
-    alerting.flatten(1)
-  end
-
-  # @return [Array<Array<String>>]
-  def tags_from_response(*path)
-    hash = response.dig(*path) || {}
-    hash.keys.map { |k| k.to_s.split(",").sort }
+    groups = response.dig(:state, :groups) || {}
+    groups.keys.flat_map { |k| k.to_s.split(",") }
```

Comment:
Do you want to sort them just to keep the behavior same?

Reply:
I added sorting to make silencing work, so removing it now since it has no benefit